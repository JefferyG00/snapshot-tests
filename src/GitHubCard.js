import React from 'react';
import Card from 'react-bootstrap/Card';
import Button from 'react-bootstrap/Button';

const GitHubCard = () => {
  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src="https://avatars.githubusercontent.com/u/147457991?v=4" />
      <Card.Body>
        <Card.Title>JefferyG00</Card.Title>
        <Card.Text>
          i game more than i study LOL
        </Card.Text>
        <Button variant="primary" href="https://github.com/JefferyG00" target="_blank">Go to GitHub</Button>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
