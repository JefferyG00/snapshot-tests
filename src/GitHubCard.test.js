import React from 'react';
import renderer from 'react-test-renderer';
import GitHubCard from './GitHubCard';

test('GitHubCard component snapshot test', () => {
  const component = renderer.create(<GitHubCard />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
