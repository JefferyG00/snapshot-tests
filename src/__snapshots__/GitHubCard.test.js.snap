// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard component snapshot test 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="https://avatars.githubusercontent.com/u/147457991?v=4"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      JefferyG00
    </div>
    <p
      className="card-text"
    >
      i game more than i study LOL
    </p>
    <a
      className="btn btn-primary"
      disabled={false}
      href="https://github.com/JefferyG00"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
      target="_blank"
    >
      Go to GitHub
    </a>
  </div>
</div>
`;
